imod_coupler/drivers/metamod/metamod.py

```diff
@@ -14,12 +14,13 @@
 from scipy.sparse import csr_matrix, dia_matrix
 from xmipy import XmiWrapper
 
+from imod_coupler.drivers.driver import Driver
 from imod_coupler.utils import create_mapping
 
 logger = logging.getLogger(__name__)
 
 
-class MetaMod:
+class MetaMod(Driver):
```

Comment:
I think it would be good to add some docstrings here of what the "MetaMod" object takes care off:
It provides a driver to couple MetaSWAP to Modflow 6.

Reply:
Fixed